src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -754,6 +756,105 @@ void ThermostatAttrAccess::OnFabricRemoved(const FabricTable & fabricTable, Fabr
     }
 }
 
+void EmitSetpointEvent(const ConcreteAttributePath & attributePath, SystemModeEnum systemMode,
```

Comment:
```suggestion
void GenerateSetpointEvent(const ConcreteAttributePath & attributePath, SystemModeEnum systemMode,
```
and similar elsewhere.